scripts/postinst.sh

```diff
@@ -0,0 +1,6 @@
+echo "**** Fixing up the permission ****"
+/opt/zimbra/libexec/zmfixperms
+if [ -d /opt/zimbra/conf/templates/ ]; then
```

Comment:
can we move this to zmfixperms script?
that way it will be done in a generic way

Reply:
This is not possible in patch. As zmfixperms is part of core package, which is not split enough to ship part of patch.

Reply:
ok